content/applications/services/helpdesk/overview/help_center.rst

```diff
@@ -0,0 +1,431 @@
+===========
+Help center
+===========
+
+Odoo *Helpdesk* integrates with the *Forums*, *eLearning*, and *Knowledge* apps to create the *help
+center*.
+
+.. image:: help_center/help-center-enable-features.png
+   :align: center
+   :alt: Overview of the settings page of a team emphasizing the help center features.
+
+The *help center* is a centralized location where teams and customers can search for (and share)
+detailed information about products and services.
+
+.. note::
+   In order to activate any of these features on a *Helpdesk* team, (*Forums*, *eLearning*, or
+   *Knowledge*), the :guilabel:`Visibility` of the team has to be set to :guilabel:`Invited portal
+   users and all internal users`. See :doc:`Getting Started
+   </applications/services/helpdesk/overview/getting_started>` for more information on *Helpdesk*
+   team settings and configuration.
+
+.. warning::
+   Since all of the *help center* features require integration with other applications, enabling any
+   of them may result in the installation of additional modules (or applications).
+
+   Installing a new application on a *One-App-Free* database will trigger a 15-day trial. At the end
+   of the trial, if a paid subscription has not been added to the database, it will no longer be
+   active or accessible.
+
+Knowledge
+=========
+
+Odoo's *Knowledge* application is a collaborative library where users can store, edit, and share
+information. The *Knowledge* app is represented throughout the database by a *book* icon.
+
+.. image:: help_center/help-center-knowledge-book-icon.png
+   :align: center
+   :alt: View of a message in Helpdesk focusing on the Knowledge book icon.
+
+Enable knowledge on a Helpdesk team
+-----------------------------------
+
+To enable the *Knowledge* feature on a *Helpdesk* team, go to :menuselection:`Helpdesk -->
+Configuration --> Teams` and select a team, or create a :doc:`new one
+</applications/services/helpdesk/overview/getting_started>`.
+
+When a team has been selected (or created), Odoo displays that team's detail form.
+
+On the team's detail form, scroll down to the :guilabel:`Self-Service` section. Click the box next
+to :guilabel:`Knowledge` to activate the *Knowledge* feature. When clicked, a new field labeled
+:guilabel:`Article` appears.
+
+Clicking the :guilabel:`Article` field reveals a drop-down menu. At first, there is only one option
+in the drop-down menu titled :guilabel:`Help`, which Odoo provides by default. Select
+:guilabel:`Help` from the drop-down to choose this article.
+
+.. tip::
+   To create a new article, go to :menuselection:`Main Dashboard --> Knowledge`. In the *Knowledge*
+   app, hover the cursor next to the :guilabel:`Workspace` section header, located in the left
+   sidebar. Moving the cursor there reveals a hidden :guilabel:`+ (plus sign)` icon.
```

Comment:
Is there a use case you've seen where **menuselection** instructions** include "Main Dashboard?
I ask because I've never seen it before, and the **menuselection** case followed by *italics* when we're actively looking at that dashboard looks strange imo because the formatting sends mixed messages.

To be clear, this paragraph is *technically* correct.

However here are my suggestion for improvement around that **menuselection** tag, along with a word change (header >> heading), and using an emoji.

```suggestion
   To create a new article, go to the :menuselection:`Knowledge app`, then hover the cursor next to
   the :guilabel:`Workspace` section heading, located in the left sidebar. Moving the cursor there
   reveals a hidden :guilabel:`➕ (plus sign)` icon.
```

Reply:
I think this was the result of one too many suggested edits, I will adjust.